app/services/subscriptions/organization_billing_service.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 
 module Subscriptions
-  class BillingService < BaseService
```

Comment:
Renaming (or changing args of) the job class will make all already enqueued jobs fail.

Maybe it's fine because this queue is usually empty (process quucky and I'm sure nobody upgrades of the first of the month but I want to confirm we're ok with it.

Keeping the job around is not necessarily a great solution either.

Reply:
Yes indeed. Since the job is retried every hour by the clock anyway, I think in that specific case, we can skip the "safe renaming approach"